src/pybind/mgr/object_format.py

```diff
@@ -529,3 +529,76 @@ def _format_response(*args: Any, **kwargs: Any) -> Tuple[int, str, str]:
         # on the ceph cli/api
         setattr(_format_response, "extra_args", {"format": str})
         return _format_response
+
+
+class ErrorResponseHandler:
```

Comment:
This is a class specifically to help deal with turning python exceptions defined here in object_format.py explicitly into mgr response tuples.  It can stack with other decorators, but in most cases the other decorators will have already caught by the decorator types in object_format.py.  It can be used as a base class or standalone.  It mainly exists to help transition code that did special one off things to use more pythonic exception handling.  Let me know if that was sufficient or if you need more.

Reply:
no, makes sense